docs/current/README.md

```diff
@@ -144,11 +144,11 @@ Log:
 - `ServerClient`: Configuration for vaquero-agent's ServerAPI client.
 - `ServerAPI`: The vaquero-server api, used by vaquero agents.
 - `AssetServer`: The asset server for Vaquero agent used by each booting machine to get unattended scripts and kernels.
-- `SavePath`: The Vaquero server location to save local configurations on disk.
+- `SavePath`: The Vaquero server location to save local configurations (data models) on disk.
```

Comment:
SavePath applies to both servers and agents.  it could be used as your "persistent" storage if no etcd details provided.

Can you follow through and just see where else we use savepath.